### DISCUSSION TITLE
Export results from find tab

### DISCUSSION THREAD

**Discussion by truemassimo** [WerWolv/ImHex#1176] [Get Help]
I need to export the results of a search, offset and founded values, it is, also, not possible select results (in particular offsets) and make a copy to past them somewhere.
Any kind of help will be very appreciated

Thanks

**Comment by iTrooz** [comment]
Hey, if you want to copy a value, you can right click on it and click "Copy value"

I agree that this isn't very convenient for bulk-copying tho.
Would you mind telling us how you want to copy these values, and what format you would expect, so we can know the best way we should implement a bulk-copy operation ?

**Reply by truemassimo** [reply]
Thanks for prompt replay,
I have made a binary search of a pattern ( 010017000000??00000005????????00000005???????? ), on a big file (2 TB), and I have found 170 items, so copy one by one is difficult, and when I try to copy I can't copy offset of the item founded (thats what I try to find) but only the founded value.

**Reply by iTrooz** [reply]
What do you think a "copy all" button (to copy all offsets and values) should copy to your clipboard ?

**Reply by truemassimo** [reply]
It would be great

**Comment by paxcut** [comment]
Just to make sure it is clear, but correct me if I'm wrong, I think iTrooz is asking about the specific format of the contents that you would expect being copied to the clipboard when a "copy all" button is pressed.

**Reply by truemassimo** [reply]
I need both offsets and values, in a CSV like format. A "copy all" button to be able to paste on a notepad and save it, it would be a very good solution. I need to parse the obtained list of values to understand if there is a particular structure (in particular the offsets are very important because the distance between them identifies the structure I'm searching for).

**Reply by MobCat** [reply]
Agree with this. and export to csv or tsv (for unicode) button would be nice.
Offset, Size, Value
0x00000002, 23, Hello world!

You can right click -> Copy Value on a value to copy a single value which is fine if that's what you wanna do.
You can ctrl+click to select multiple values, however when you right click -> Copy value it only copies the value you clicked on, not all selected values.
